src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs

```diff
@@ -258,6 +270,79 @@ private bool GetIsInheritDoc(XPathNavigator nav, ITripleSlashCommentParserContex
             }
             return true;
         }
+        
+        private void ResolveCodeSource(XDocument doc, ITripleSlashCommentParserContext context)
+        {
+            foreach (XElement node in doc.XPathSelectElements("//code"))
+            {
+                var source = node.Attribute("source");
+                if (source == null || string.IsNullOrEmpty(source.Value))
+                {
+                    continue;
+                }
+
+                if (context.Source == null || string.IsNullOrEmpty(context.Source.Path))
+                {
+                    Logger.LogWarning($"Unable to get source file path for {node.ToString()}");
+                    return;
+                }
+
+                var region = node.Attribute("region");
+
+                var path = source.Value;
+                if (!Path.IsPathRooted(path))
+                {
+                    var directory = Path.GetDirectoryName(context.Source.Path);
+                    path = Path.Combine(context.Source?.BasePath ?? ".", directory, path);
```

Comment:
Path is not relative to docfx.json, https://github.com/dotnet/docfx/blob/master/src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/VisitorHelper.cs#L150, it is the local file path fetched from SyntaxTree